hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneHAClusterImpl.java

```diff
@@ -442,7 +457,169 @@ private void initHAConfig(int basePort) throws IOException {
         conf.setInt(omRatisPortKey, port + 4);
       }
 
-      conf.set(omNodesKey, omNodesKeyValue.substring(1));
+      conf.set(omNodesKey, String.join(",", omNodeIds));
+    }
+  }
+
+  /**
+   * Bootstrap new OMs and add them to existing OM ring.
+   * @param numNewOMs number of new OMs to bootstrap
+   * @return list of new OMs nodeIds
+   */
+  public List<String> bootstrapOzoneManagers(int numNewOMs) throws IOException,
```

Comment:
Why is the number of newOMs an argument? I was expecting this to read based on the new HA configuration

Reply:
Giving numNewOMs are parameter to test different scenarios - adding OMs one by one and adding two OMs together. This is the function which generates the new HA configuration for the new nodes.